src/rt/test/systematic.h

```diff
@@ -303,8 +313,8 @@ namespace Systematic
 #endif
 
   private:
-    std::ostream& o;
```

Comment:
Slightly off-topic, but this direct dependency on iostream is the thing that makes it hardest to run Verona bare-metal.  It would be nice to encapsulate this somewhere (though possibly this class is sufficient).

Reply:
I think this class is the right place to encapsulate that.  This could then be a ring buffer out of an enclave, or other minimal system. I think everything higher level can be unchanged (probably).